docs/config.md

```diff
@@ -76,8 +76,8 @@ Notes:
 | lon_name              | string  | Yes       | n/a     |                                                                                          |
 | time_name             | string  | Yes       | n/a     |                                                                                          |
 | nfiles                | integer | Yes       | -1      |                                                                                          |
-| transformations       | integer | No        | 1       | no transform = 1, quantile mapping = 2, log transform = 3, cube root = 4, fifth root = 5 |
-| input_transformations | integer | No        | 1       | no transform = 1, quantile mapping = 2, log transform = 3, cube root = 4, fifth root = 5 |
+| transformations       | integer | No        | 0       | no transform = 0, quantile mapping = 1, log transform = 2, cube root = 3, fifth root = 4 |
+| input_transformations | integer | No        | 1       | no transform = 0, quantile mapping = 1, log transform = 2, cube root = 3, fifth root = 4 |
```

Comment:
You are right. That should have actually made it into #19 but I'll sneak it in here.